presto-hive/src/main/java/com/facebook/presto/hive/util/SortBuffer.java

```diff
@@ -110,8 +112,6 @@ public void flushTo(Consumer<Page> consumer)
             positionIndex[i] = pageSorter.decodePositionIndex(addresses[i]);
         }
 
-        PageBuilder pageBuilder = new PageBuilder(types);
```

Comment:
maybe replace this with an assertion that the page builder is empty `verify(pageBuilder.isEmpty);`